go.mod

```diff
@@ -3,3 +3,5 @@ module github.com/edgexfoundry/app-service-configurable
 go 1.16
 
 require github.com/edgexfoundry/app-functions-sdk-go/v2 v2.0.1
+
+replace github.com/edgexfoundry/app-functions-sdk-go/v2 => ../app-functions-sdk-go
```

Comment:
this seems like it was checked in by mistake

Reply:
Nope, this is way the PR is in draft until the SDK PR is merged and have a new tag with the new capability.